testdata/p4_16_errors/actions-with-duplicate-control-plane-names1.p4

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
@chrispsommers @smolkaj This is one of the two new P4 test programs I would recommend you focus your review on.  The comments indicate which parts of the program I expect should give errors, and you can look at the corresponding testdata/p4_16_errors_output/actions-with-duplicate-control-plane-names1.p4-stderr file here to see that it does give errors for those: https://github.com/p4lang/p4c/pull/4975/files#diff-30d824dc2073bd520ea90c54fc52935bad924ddc326b1ab76e225e9c94bacae5